addons/rook/1.6.11/operator/resources.yaml

```diff
@@ -15,6 +15,7 @@ spec:
     plural: cephblockpools
     singular: cephblockpool
   scope: Namespaced
+  preserveUnknownFields: false
```

Comment:
Some of the CRDs have moved from v1beta1 to v1. And `preserveUnknownFields` is deleted in v1. If we define it again and mark as `false`, it will use the default schema changes which is `x-kubernetes-preserve-unknown-fields: true`
Pretty much followed what is suggested in this thread.
https://github.com/kubernetes-sigs/controller-tools/issues/476#issuecomment-691519936

Reply:
is this added manually, or will running the generation process again drop this change?

If it's manual, can we add it to the generation scripts?